src/main/java/persistence/sql/QueryBuilder.java

```diff
@@ -0,0 +1,112 @@
+package persistence.sql;
+
+import jakarta.persistence.*;
+
+import java.lang.reflect.Field;
+import java.util.*;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public abstract class QueryBuilder {
+    protected static final String BLANK = " ";
+    protected static final String COMMA = ",";
+    protected static final String SINGLE_QUOTE = "'";
+    protected static final String NULL = "null";
+
+    protected Class<?> entity;
+    protected final Columns idColumns = new Columns();
+    protected final Columns columns = new Columns();
```

Comment:
```suggestion
    protected final Columns idColumns = new Columns();
    protected final Columns columns = new Columns();
    
    protected Class<?> entity;
```

불변 필드 값에 대한 부분도 구분지어주면 더 좋을 것 같습니다!